src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      >
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__stars stars">
+        <div class="stars-stars stars-stars--4">
+          <div class="stars-stars__star"></div>
+          <div class="stars-stars__star"></div>
+          <div class="stars-stars__star"></div>
+          <div class="stars-stars__star"></div>
+          <div class="stars-stars__star"></div>
+        </div>
+        <div class="stars__amount">
+          Reviews: 5
+        </div>
+      </div>
```

Comment:
```suggestion
      <div class="card__reviews">
        <div class="stars stars--4 card__stars">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <div class="card__reviews-amount">
          Reviews: 5
        </div>
      </div>
```
Make it work, if you have any questions feel free to ask in fe_chat1 in Slack

Reply:
thanks, i was a little bit tired when i decided to name classes in so weird way))